src/sentry/api/endpoints/organization_relay_usage.py

```diff
@@ -14,7 +14,7 @@
 class OrganizationRelayUsage(OrganizationEndpoint):
     owner = ApiOwner.OWNERS_INGEST
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PRIVATE,
```

Comment:
This one should be public in my opinion. A user can see a list of their relays in the UI, so why not allow them to query them via the API?

Reply:
That's fair, yeah. I made this public in [eec8e7a](https://github.com/getsentry/sentry/pull/61828/commits/eec8e7a49dfecc6a652d34cec407f07ddffe1170).

Reply:
Thanks.